src/Microsoft.AspNetCore.Authentication/Events/RemoteFailureContext.cs

```diff
@@ -11,6 +11,7 @@ namespace Microsoft.AspNetCore.Authentication
     /// </summary>
     public class RemoteFailureContext : HandleRequestContext<RemoteAuthenticationOptions>
     {
+        [Obsolete]
```

Comment:
Should this have some text to go along with the warning?

Reply:
*shrug* we don't really expect anyone to use these directly.

Reply:
Why bother with obsolete at all, its not incorrect to use this ctor is it?

Reply:
It's obsolete as the first step towards getting rid of it, it's not used anymore.